app/views/layouts/application.html.erb

```diff
@@ -18,6 +18,9 @@
           localStorage.removeItem('previous-search-url');
           window.location = searchUrl;
       }
+
```

Comment:
can we scope this to the window.SCP object?

Reply:
Addressed in 034ff9d0.